client/express-checkout/index.js

```diff
@@ -30,6 +32,7 @@ jQuery( ( $ ) => {
 	}
 
 	const publishableKey = wcpayExpressCheckoutParams.stripe.publishableKey;
+	const quantityInoutSelector = '.quantity .qty[type=number]';
```

Comment:
I guess this is a typo? If so, we need to replace it in the references as well.

```suggestion
	const quantityInputSelector = '.quantity .qty[type=number]';
```

Reply:
👀 Good catch! Fixed https://github.com/Automattic/woocommerce-payments/pull/8987/commits/17c03e6ae2d03d3e74080ca40d23f05543a3fcbb.